app/stac_api/validators.py

```diff
@@ -216,3 +223,306 @@ def validate_asset_multihash(value):
             message=_('Invalid multihash value; %(error)s'),
             params={'error': error}
         )
+
+
+class ValidateSearchRequest:
```

Comment:
This class is very good however it should be part of the `validators_serializer.py` module to not messup with the django ValidationError and the rest_framework ValidationError. The validators.py module should be only for validators that can be used by models and/or serializers. For views/serializers validator only we should use validators_serializer.py

Reply:
ok